docs/networks.md

```diff
@@ -0,0 +1,20 @@
+# Using Networks
```

Comment:
Also, the other documentation files are written using "you" instead of "we". This needs to use the same perspective.